.github/workflows/ci_ubuntu.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration with Ubuntu
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        source ci/ci_ubuntu.sh
+        dependencies
+
+    - name: Style
+      run: |
+        source ci/ci_ubuntu.sh
+        style
```

Comment:
It would be good if the style test can be run as a separate job. Because it only needs to run on one python version. Also, style requires much less dependencies, so it should be much faster for that job to finish without people having to wait to see if there are PEP8 issues.

Finally, looking at the failures in the CI, it all has to do with imports. If this is a thing in plyer, maybe we can disable that check?

Reply:
Respecting running the style test as a separate job -> it's done.

If I recall correctly, most of the import warning are because of the PY2 support, that will be removed soon. I don't know if the other imports are explicitly needed in the place they are.

By the way, considering that Kivy checks the style with flake8, should we do the same in plyer? Currently the checks are done with pylint and pycodestyle.

Reply:
>If I recall correctly, most of the import warnings are because of the PY2 support, that will be removed soon.

Is there a issue about dropping py2 support? If so we should make it clear that py2 support will be dropped after this upcoming release. And we can silence those py2 warning for now until it can be removed.

>By the way, considering that Kivy checks the style with flake8, should we do the same in plyer?

That sounds like a good idea.